.github/workflows/ci-dotnet-action.yml

```
@@ -0,0 +1,25 @@
+name: .NET
+
+on:
+  push:
+    branches:
+     - main
+  pull_request:
+    branches:
+     - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore ./backend/Car-Detailing-Backend/Car-Detailing-Backend.csproj
+    - name: Build
+      run: dotnet build ./backend/Car-Detailing-Backend/Car-Detailing-Backend.csproj --no-restore
```

You could set default working directory in this yml like I do in file for frotend.
```yamls
        defaults:
            run:
                working-directory: backend/Car-Detailing-Backend
```

```suggestion
    - name: Restore dependencies
      run: dotnet restore ./Car-Detailing-Backend.csproj
    - name: Build
      run: dotnet build ./Car-Detailing-Backend.csproj --no-restore
```